src/modules/__tests__/nav.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { displayCount } from '../nav.js';
+import mockNav from '../__mocks__/nav.mock.js';
+
+test('Add x items and check count', () => {
```

Comment:
- [ ] It would be better if we write unit test for our function for displaying counters. This function like for example; `getCommentCount` needs to be tested :man_technologist:

Reply:
That file is responsible for testing the counter on the homepage. The getCommentCount function does indeed get tested, only in the file popup.test.js. Both tests are working